lib/matplotlib/pyplot.py

```diff
@@ -3787,4 +3791,60 @@ def spectral():
     if im is not None:
         im.set_cmap(cm.spectral)
 
+
+# This function was autogenerated by boilerplate.py.  Do not edit as
+# changes will be lost
+def magma():
```

Comment:
There are auto generated so all that is done here is to regenerate pyplot with boilerplate.py 

If you want to change that I suggest raising  a new issue.